DashAI/front/src/components/experiments/PrepareDatasetStep.jsx

```diff
@@ -81,10 +111,38 @@ function PrepareDatasetStep({ newExp, setNewExp, setNextEnabled }) {
 
   useEffect(() => {
     getDatasetInfo();
+    getTaskRequirements();
   }, []);
+
+  const parseListOfStrings = (stringsList) => {
+    return stringsList.join(" or ");
+  };
   return (
     <React.Fragment>
-      {!loading ? (
+      {!requirementsLoading && (
```

Comment:
I would leave this alert independent of loading. Maybe just show it always.

Reply:
if I leave alert independent of loading the task_requirements dictionary would be empty when rendering so (in the last commit) I left a dummy dict in task_requirements definition, with this I could delete requirements loading so we don't have to worry about two loadings getting out of sync.